schemaregistry/rest_service.go

```diff
@@ -172,6 +173,9 @@ func configureTLS(conf *Config, tlsConf *tls.Config) error {
 		if err != nil {
 			return err
 		}
+		if tlsConf.RootCAs == nil {
```

Comment:
Good find!

@emasab @rayokota From what I can see this code never worked, and was thus never tested. Can we add a test case?

Also, the tls docs says RootCAs will default (if nil) to the system CAs, which makes me wonder if the suggested fix is correct, or if AppendCertsFromPEM.. call isn't correct?